gamestonk_terminal/comparison_analysis/ca_controller.py

```diff
@@ -107,6 +109,8 @@ def print_help(self):
         print("   performance   brief performance comparison")
         print("   technical     brief technical comparison")
         print("")
+        print("   po            portfolio optimzation from selected tickers")
```

Comment:
To keep consistency, I would do:
```
print("  > po            portfolio optimzation from selected tickers")
```
where `>` means that it's a menu.

Reply:
no idea how I never caught that trend

Reply:
I think it's only on the fundamental analysis menu, to dive into > av and > fmp.

I think it's nice to have a slight distinction compared normal commands. Probably even the main menu should have them..